app/src/main/res/values/styles.xml

```diff
@@ -464,4 +464,12 @@
   <style name="TextViewCenterHorizontal" parent="TextView.Common1">
     <item name="android:gravity">center_horizontal</item>
   </style>
+
+  <!-- ShapeableImageView -->
+  <!-- context for ShapeableimageView https://howtodoandroid.com/shapeableimageview-material-components-android/
+       -->
+  <!-- Here key to the overlay behavior is the empty parent attribute. This is what causes the overlay behavior -->
```

Comment:
Please combine these three comments into one and make sure to use full sentence structures (except for the first bit, that can just be 'ShapeableImageView').

The last bit could also probably be a bit clearer, e.g.: 'Note that the parent attribute is intentionally empty because if enables the overlay behavior for the shape due to ...' (note that the last bit needs to be filled in--the current text doesn't actually explain why this works).

Finally, please move the context line to the end (usually contexts are after the rest of the explanation to provide further reading).

Reply:
Done